packages/gui/src/components/nfts/detail/NFTDetailV2.tsx

```diff
@@ -139,15 +231,15 @@ export default function NFTDetail() {
                     </Typography>
                   </Flex>
                 )}
-                {(nft?.editionTotal ?? 0) > 1 && (
+                {(nft?.seriesTotal ?? 0) > 1 && (
                   <Flex flexDirection="column" gap={1}>
                     <Typography variant="h6">
-                      <Trans>Edition Number</Trans>
+                      <Trans>Series Number</Trans>
                     </Typography>
 
                     <Typography>
                       <Trans>
-                        {nft.editionNumber} of {nft.editionTotal}
+                        {nft.seriesNumber} of {nft.seriesTotal}
```

Comment:
Please revert - `editionNumber` and `editionTotal` are correct

Reply:
Done.